docs/src/pages/[platform]/getting-started/installation/dependencies.vue.mdx

```diff
@@ -25,3 +25,21 @@ yarn add aws-amplify @aws-amplify/ui-vue
   documentation](https://github.com/aws-amplify/amplify-ui/tree/legacy/legacy/amplify-ui-vue)
   please click here.
 </Alert>
+
+### Install as a global plugin (Optional)
```

Comment:
This was tacked onto the Troubleshooting fragment for Vue, but seems like it's meant to be part of the Installation page. Should I make this a new fragment that only exists for Vue or tack it on here?

Reply:
This seems more like a usage fragment than an installation fragment to me, since it shows how to use it in an application.